torch/csrc/jit/passes/tensorexpr_fuser.cpp

```diff
@@ -463,6 +486,29 @@ struct TensorExprKernel {
             });
       } break;
 
+      case aten::min: {
+        return ComputeTwoOperand(
+            "aten_min", n, [](const Expr& lhs, const Expr& rhs) {
+              return Min::make(lhs, rhs, true);
+            });
+      } break;
+
+      case aten::max: {
+        return ComputeTwoOperand(
+            "aten_max", n, [](const Expr& lhs, const Expr& rhs) {
+              return Max::make(lhs, rhs, true);
+            });
+      } break;
+
+      case aten::clamp: {
+        return ComputeThreeOperand(
+            "aten_max",
+            n,
+            [](const Expr& in, const Expr& min, const Expr& max) {
+              return Max::make(Min::make(in, max, false), min, false);
```

Comment:
Can you verify the NaN-behavior of clamp vs the eager mode kernels?

Reply:
Yeah definitely. Was trying to figure it out by reading code but it’s probably faster to just test.

Reply:
Ok, I've changed the llvm codegen to match eager mode.  It's not theoretically pure since it's conditional propagation, but, it matches.